### ISSUE TITLE
Integration example is no longer working

### ISSUE THREAD

**Issue by ssaavedra** [osohq/oso-django-integration#1]
Not sure if this is related to changes in Oso that have not been made into this integration example or if the documentation is currently just hard to follow, but I cannot get this example to work, and since the `requirements.txt` don't have a pinned version of the `oso` and `django_oso` libraries, so I have no idea if this is a regression or how to track the source of this issue.

Is this integration still in scope to work, or is the project unmaintained at this stage?

I've run this with `POLAR_LOG=trace` and got the following two lines:

```
[oso][trace]   Query(allow(<django.contrib.auth.models.AnonymousUser object at 0x7f91d2b31760> TYPE `UNKNOWN`, "GET", <WSGIRequest: GET '/'> TYPE `UNKNOWN`))
[oso][info]   QUERY RULE: allow(<django.contrib.auth.models.AnonymousUser object at 0x7f91d2b31760> TYPE `UNKNOWN`, "GET", <WSGIRequest: GET '/'> TYPE `UNKNOWN`), BINDINGS: {}
```

![imagen](https://user-images.githubusercontent.com/581152/170506080-978ce440-f7c1-426e-a5f6-aa1afb58e6de.png)

<details><summary>Complete stack trace</summary>

```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8001/

Django Version: 3.1
Python Version: 3.8.10
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'expenses.apps.ExpensesConfig']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'expenses.authorization.get_user',
 'django_oso.middleware.RequireAuthorization',
 'django_oso.middleware.RouteAuthorization']



Traceback (most recent call last):
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/django_oso/middleware.py", line 77, in middleware
    authorize(request, resource=request)
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/django_oso/auth.py", line 34, in authorize
    authorized = Oso.is_allowed(actor, action, resource)
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/oso/oso.py", line 67, in is_allowed
    next(self.query_rule("allow", actor, action, resource))
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/polar/polar.py", line 133, in query
    for res in Query(query, host=host, bindings=bindings).run():
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/polar/query.py", line 48, in run
    ffi_event = self.ffi_query.next_event()
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/polar/ffi.py", line 164, in next_event
    event = read_c_str(self.check_result(event))
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/polar/ffi.py", line 192, in check_result
    return check_result(result, self.enrich_message)
  File "/home/ssaavedra/projects/public/oso-django-integration/venv/lib/python3.8/site-packages/polar/ffi.py", line 225, in check_result
    raise error

Exception Type: PolarRuntimeError at /
Exception Value: Query for undefined rule `allow`
	Get help with Oso from our engineers: https://help.osohq.com/error/PolarRuntimeError
```
</details>

Any help would be much appreciated.

**Comment by ssaavedra** [comment]
I've just went through several pip releases and found out that `oso==0.24.0` is the latest one to throw 403 Forbidden on the main page instead of this `PolarRuntimeError`.

**Comment by gj** [comment]
Hi @ssaavedra,

Sorry for taking so long to get back to you. I believe this is due to a breaking change after 0.24 that requires policies to contain at least one `allow(actor, action, resource)` entrypoint rule. The simplest form of that rule is:

```py
allow(actor, action, resource) if has_permission(actor, action, resource);
```

Would you be up for submitting a PR to make that change to this repo?

**Comment by ssaavedra** [comment]
I did not have the time budget to get back to this. It seems not related to that, but to the fact that you have to qualify the registered classes now. Sent a PR your way.
